docs/integrations/builtin/app-nodes/n8n-nodes-base.actionnetwork.md

```diff
@@ -1,9 +1,16 @@
 # Action Network
 
-[Action Network](https://actionnetwork.org/) is an open platform that empowers individuals and groups to organize for progressive causes.
+
+
+The [Action Network](https://actionnetwork.org/){:target="_blank" .external-link} node allows you to automate work in Action Network and integrate Action Network with other applications. n8n has built-in support for a wide range of Action Network features, which includes creating, updating, and deleting events, people, tags, and signatures. 
+
+On this page, you'll find a list of operations the Action Network node supports, and links to more resources.
 
 !!! note "Credentials"
-    You can find authentication information for this node [here](/integrations/builtin/credentials/actionnetwork/).
+    Refer to the [Action Network credentials](/integrations/builtin/credentials/actionnetwork/) for guidance on setting up authentication. 
+
+!!! note "Examples and templates"
+    For example, usage and templates to help you get started, take a look at n8n's [Action Network integrations](https://n8n.io/integrations/action-network/) list.
```

Comment:
This should use the external link format, as it goes out of the docs to the main website.